docs/how-to-guides/software-management/software-support.md

```diff
@@ -49,4 +72,15 @@ Prioritization meetings are typically a loosely guided discussion in which the t
 1. Ask for any closing remarks
 
 ??? Note "Issue deferral"
-    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
\ No newline at end of file
+    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization. 
+
+
+## Continuous Support 
+
+Internal developers have time to perform basic support tasks throughout the week. We refer to this as continuous support, as it's support time between dedicated sprints. This is to stay on top of GitHub notifications. 
```

Comment:
Is the intention that continuous support is only for 'keeping up on notifications'. What does that mean? Are things fixed during continuous support? Or is this just 'reading email' and taking notes.

Reply:
bruh you told me to take that part out.

Reply:
I know this merged. I wanted to respond the the "bruh". I know I did. And almost everything was taken out (which is appreciated!). What remained was the 'to stay on top of GitHub notifications' which was still something that I fear external people would latch onto. 

Next time, I'll strive to give a longer statement along the lines of 'this is headed in a great direction, but consider removing the final sentence'. The way I responded above clearly was neither helpful nor communicative enough. 

Apologies for the confusion. Hopefully this late reply gives some additional context to my comment.